pkg/collector/corechecks/snmp/fetch/fetch_column.go

```diff
@@ -52,7 +52,7 @@ func fetchColumnOidsWithBatching(sess session.Session, oids map[string]string, o
 // fetchColumnOids has an `oids` argument representing a `map[string]string`,
 // the key of the map is the column oid, and the value is the oid used to fetch the next value for the column.
 // The value oid might be equal to column oid or a row oid of the same column.
-func fetchColumnOids(sess session.Session, oids map[string]string, bulkMaxRepetitions uint32) (valuestore.ColumnResultValuesType, error) {
+func fetchColumnOids(sess session.Session, oids map[string]string, bulkMaxRepetitions uint32, useGetNext bool) (valuestore.ColumnResultValuesType, error) {
```

Comment:
For the sake of readability, an enum, instead of a boolean, might make the code even more explicit. Example:

Seeing a `false` here:
`columnResults, err := fetchColumnOidsWithBatching(sess, oids, config.OidBatchSize, config.BulkMaxRepetitions, false)`

is not very easy to interpret (is it getbulk? is it getnext?) 
Maybe, we could have a "strategy" enum like this?

```
type strategy int
const (
   UseGetNext strategy = iota
   UseGetBulk
)
...

results, err := fetchColumnOids(sess, oidsToFetch, bulkMaxRepetitions, UseGetNext) // enum instead of bool
```

Reply:
@gmarabout good idea. fixed in last 2 commits https://github.com/DataDog/datadog-agent/pull/9704/commits/5e135b2cba48670a9b33e2848109d3a78c142869 and https://github.com/DataDog/datadog-agent/pull/9704/commits/b9eb91503563fcafa7031e2840c1d7b369dca832